src/Controller/ApplePay.php

```diff
@@ -464,6 +465,15 @@ public function process_standard_payment($order, $amount, $customer_id, $workflo
 				]
 			];
 
+			if (PayplugWoocommerceHelper::is_page_with_blocks()) {
```

Comment:
can you do: if is_cart: 
    $payment_data['metadata']['woocommerce_block'] = "CART";
else
	$payment_data['metadata']['woocommerce_block'] = "CHECKOUT";

because imagine you're in the order_review page, or stuff like that

Reply:
the order review page is not blocks, so we shouldn't send this metadata